tools/README.md

```diff
@@ -0,0 +1,65 @@
+# Using prebuilt images with gRPC OSS benchmarks
+
+The scripts in this folder:
+* Build the images with the worker executables compiled and embedded.
+* Push the images to specified registry.
+* Delete the images from specified registry.
+
+The helper scripts here should be specifically used with GCR. The predominate drive of developing these
+scripts are to assist continuous build. The scripts could also help with manual usage
+of prebuilt images if working with GCR. If another image registry is chosen, the prebuilt workers Dockerfiles are available. The root directory of the Dockerfiles could be find [test-infra/containers/pre_built_images](test-infra/containers/pre_built_images).
+
+## Build and push images
+
+The script [prepare_prebuilt_workers.go](pre_built_workers/prepare_prebuilt_workers.go) 
+builds images and pushes them to a user specified registry. For example, the following shows this process (building and pushing prebuilt images) for cxx and go workers:
+```
+go run ./prebuilt_workers/prepare_prebuilt_workers/prepare_prebuilt_workers.go \
+ -l cxx:master \
+ -l go:master \
+ -p gcr.io/<grpc-project>/pre_built_workers \
```

Comment:
I find it a bit odd that only the -r flag needs a trailing slash. We can resolve that in a follow-up PR by adding the slash if the value is not empty and does not end in a slash.